lib/pubsub_client/subscriber.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'google/cloud/pubsub'
+
+module PubsubClient
+  class Subscriber
+    DEFAULT_CONCURRENCY = 8
+
+    # @param subscription [Google::Cloud::PubSub::Subscription]
+    def initialize(subscription)
+      @subscription = subscription
+    end
+
+    # @param concurrency [Integer] - The number of threads to run the subscriber with.
+    # @param auto_ack    [Boolean] - Flag to acknowledge the Pub/Sub message. A message must be
```

Comment:
FYI - While the spacing you've added makes this nice to read now, adding a new parameter with a longer name or changing the name of one of the existing parameters can force you to re-indent the entire comment, which adds extra noise to a PR.